alloc-customization-point.html

```diff
@@ -90,16 +95,25 @@ <h2>tl;dr</h2>
     std::pmr::polymorphic_allocator&lt;&gt;&gt;::type>
 : BloombergLP::bslma::UsesBslmaAllocator&lt;TYPE&gt;::type
 {
-  static BloombergLP::bslma::Allocator* rebind(std::pmr::polymorphic_allocator&lt;&gt; a)
+  static BloombergLP::bslma::Allocator* convert(std::pmr::polymorphic_allocator&lt;&gt;a)
     {
       return dynamic_cast&lt;BloombergLP::bslma::Allocator*&gt;(a.resource());
     }
 }</code></blockquote></pre>
 
+<p>We also allow for an in-class opt-in by providing a static member function
+<tt>domain_alloc_convert</tt> and a nested type <tt>domain_allocator_conversion_source</tt> (Note: names to be bikeshed by LEWG).
+
+</p>
+
 <p>
   Finally, we modify uses-allocator construction so that
-  it detects the presence of __domain_allocator_traits&lt;T, Alloc&gt;::rebind
-  and calls it if it's present, and uses the result as an allocator.
+  it detects the presence of <tt>__domain_allocator_traits&lt;T, Alloc&gt;::convert</tt>
```

Comment:
This description should also mention that we try the nested domain_alloc_convert member, right?

Reply:
Hmm. On the other hand, the later description seems to suggest that in that case, domain_allocator_traits::convert will just work. See below, then.

Reply:
We do only try __domain_allocator_traits<T, Alloc>::convert.  The detection mechanism binds convert to the domain_alloc_convert, and it will get invoked through that, but it will not get invoked additionally. I can mention this binding happens in the paragraph above?

Reply:
No, it's fine that it's explained below.